packages/plugin-mparticle/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@itly/plugin-mparticle",
+  "version": "1.1.0",
+  "description": "mParticle Browser Plugin",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "prepublish": "yarn run build",
+    "build": "tsc",
+    "test": "jest --runInBand"
+  },
+  "keywords": [
+    "itly",
+    "sdk",
+    "iteratively",
+    "plugin"
+  ],
+  "author": "Iteratively",
+  "license": "ISC",
+  "devDependencies": {
+    "@itly/sdk": "^1.1.0",
```

Comment:
`@itly/sdk` is here at 1.1.0 and in peerDependencies at 1.0.0. Do we actually need it in both places?

Reply:
I think so. We need it in dev to build and test, and need it as a peer in production. Unfortunately Lerna doesn't bump the peer deps automatically.